optuna/integration/mlflow.py

```diff
@@ -75,14 +75,29 @@ def objective(trial):
             Name of the metric. Since the metric itself is just a number,
             `metric_name` can be used to give it a name. So you know later
             if it was roc-auc or accuracy.
+        nest_trials:
+            Flag indicating whether or not trials should be logged as
+            nested runs. This is often helpful for aggregating trials
+            to a particular study, under a given experiment.
+        tag_study_user_attrs:
+            Flag indicating whether or not to add the study's user attrs
+            to the mlflow trial as tags.
     """
 
-    def __init__(self, tracking_uri: Optional[str] = None, metric_name: str = "value") -> None:
+    def __init__(
+        self,
+        tracking_uri: Optional[str] = None,
+        metric_name: str = "value",
+        nest_trials: Optional[bool] = False,
+        tag_study_user_attrs: Optional[bool] = False,
```

Comment:
We're not handling `None` as it seems now. Actually, it can be a mandatory parameter, just that it has a default value.
```suggestion
        tag_study_user_attrs: bool = False,
```